wiremock-spring-boot-example/src/test/java/app/HttpsOnlyTest.java

```diff
@@ -18,16 +18,16 @@
 import org.wiremock.spring.InjectWireMock;
 
 @SpringBootTest
-@EnableWireMock({@ConfigureWireMock(useHttps = true)})
-class HttpsTest {
+@EnableWireMock({@ConfigureWireMock(port = -1, httpsPort = 0)})
```

Comment:
What happens if I have `(port  = -1, httpsPort = -1)?

Reply:
Would be nice with a helpful error in this case. I'll add that.